nnbuilder/src/main/kotlin/org/hse/nnbuilder/queue/TasksQueue.java

```diff
@@ -0,0 +1,95 @@
+package org.hse.nnbuilder.queue;
+
+import java.sql.Timestamp;
+import java.util.Random;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.hse.nnbuilder.services.Tasksqueue.TaskName;
+import org.hse.nnbuilder.services.Tasksqueue.TaskStatus;
+
+@Entity
+@Table(name = "tasksqueue")
+public final class TasksQueue {
+    @Id
+    private Long taskId = new Random().nextLong();
+    @Column
+    TaskName taskName;
+    @Column
+    Long nnId;
+    @Column
+    Long dataId;
+    @Column
+    Timestamp startTime;
```

Comment:
There's some annotation to set created timestamp automatically

Reply:
However, that'd be createdTime, while startedTtime should be the moment when you actually start processing the task

Reply:
Just pointing out that this one doesn't match its name. You'll need createdTime to pop from the queue. startedTime may be helpful to monitor the queue performance or to identify task processing timeouts.